src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -1370,6 +1370,11 @@ def generate(
         else:
             decoder_input_length = 1
 
+        if kwargs is not None and "num_beams" in kwargs:
```

Comment:
Sorry, missed this from my review! Don't think we need this here since passing `num_beams` in the kwargs will also automatically get forwarded on to `.generate` (you can remove this code and try for yourself!)

In general, we shouldn't need to modify any modelling code for this PR!